boring/src/sizing/R_evap.py

```diff
@@ -0,0 +1,148 @@
+# -*- coding: utf-8 -*-
+"""
+Evaporator secion thermal resistances
+
+Created on Fri Oct 23 11:06:38 2020
+
+@author: Karsten Look
+"""
+
+from __future__ import absolute_import
+import numpy as np
+from math import pi
+
+import openmdao.api as om
+
+
+class R_evapComp(om.ExplicitComponent):
+    def initialize(self):
+        self.options.declare('num_nodes', types=int)
+
+    def setup(self):
+        nn=self.options['num_nodes']
+
+        #TODO Add units
+        self.add_input('alpha')
+        self.add_input('h_fg')
+        self.add_input('T_hp')
+        self.add_input('v_fg')
+        self.add_input('R_g')
+        self.add_input('P_v')
+        self.add_input('D_od')
+        self.add_input('r_i')
```

Comment:
We can consider converting some of these "inputs" to instead be "options". If the variable is a constant that wouldn't vary during an optimization, making it an option would reduce the number of derivatives needed.

(But there is no harm in leaving them as inputs, if you're on a roll)